js/js_test.go

```diff
@@ -64,3 +64,12 @@ func TestInt64(t *testing.T) {
 		t.Errorf("got %#v, want %#v", got, want)
 	}
 }
+
+func TestInstanceOf(t *testing.T) {
```

Comment:
A second test for the negative case would be good.

Reply:
Done.